pom.xml

```
@@ -14,7 +14,7 @@
     <version>1.42.2-SNAPSHOT</version>
     <packaging>hpi</packaging>
 
-    <url>https://wiki.jenkins-ci.org/display/JENKINS/GitHub+pull+request+builder+plugin</url>
+    <url>https://github.com/jenkinsci/ghprb-plugin/blob/master/README.md</url>
```

should just need the repo if it's the default README
```suggestion
    <url>https://github.com/jenkinsci/ghprb-plugin</url>
```